presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -281,6 +292,17 @@ public JdbcOutputTableHandle beginInsertTable(ConnectorTableMetadata tableMetada
     }
 
     private JdbcOutputTableHandle beginWriteTable(ConnectorTableMetadata tableMetadata)
+    {
+        try {
+            return createTable(tableMetadata, generateTemporaryTableName());
+        }
+        catch (SQLException e) {
+            throw new PrestoException(JDBC_ERROR, e);
+        }
+    }
+
+    protected JdbcOutputTableHandle createTable(ConnectorTableMetadata tableMetadata, String intermediateTableName)
```

Comment:
`intermediateTableName` should be `tableName` since for this method it is simply the name of the table that will be created.